src/lib/thirdwebClient.ts

```diff
@@ -1,9 +1,5 @@
 import { createThirdwebClient } from "thirdweb";
 
-export const client = process.env.THIRDWEB_SECRET_KEY
-	? createThirdwebClient({
-			secretKey: process.env.THIRDWEB_SECRET_KEY,
-		})
-	: createThirdwebClient({
-			clientId: process.env.NEXT_PUBLIC_THIRDWEB_CLIENT_ID || "",
-		});
+export const client = createThirdwebClient({
+	clientId: process.env.NEXT_PUBLIC_THIRDWEB_CLIENT_ID || "",
```

Comment:
dont you need the secret key since you're using this in SSR?

also you should throw an error if the env variable is not set.

a nice cleanup would be to centralize all the env variables in one file and export them